File: Beginner/go/1070.go
// https://www.beecrowd.com.br/judge/en/problems/view/1070

package main

import "fmt"

func main(){
	var n int
	fmt.Scan(&n)
	
	var count = 0
	for i := 0; i < 6 * 2; i++{
		if n % 2 != 0 {
			fmt.Println(n);
			count++;
		}
		n++;
	}
}

File: Beginner/cpp/1070.cpp
// https://www.beecrowd.com.br/judge/en/problems/view/1070

#include <bits/stdc++.h>
using namespace std;

int main(){
	int n;
	cin >> n;
	
	int count = 0;
	while(count < 6){
		if(n % 2 != 0){
			cout << n << endl;
			count++;
		}
		n++;
	}
	
	return 0;
}
